app/controllers/organizations/adopter_fosterer/adopted_pets/tasks_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Organizations
+  module AdopterFosterer
+    module AdoptedPets
+      class TasksController < Organizations::BaseController
+        before_action :context_authorize!
+
+        def index
+          @pet = Pet.find(params[:adopted_pet_id])
+          @tasks = @pet.tasks.where(completed: false)
+
+          respond_to do |format|
```

Comment:
Same here I think we can remove this whole block

Reply:
Done